src/api_v3.py

```diff
@@ -167,15 +167,15 @@ def manage_gateway_client_tests(msisdn):
             raise BadRequest("The 'test_start_time' field is required.")
 
         try:
-            test_start_time = datetime.fromisoformat(test_start_time)
+            test_start_time = datetime.fromtimestamp(int(test_start_time))
         except ValueError:
             raise BadRequest(
-                "Invalid 'test_start_time' format. Use ISO format (YYYY-MM-DDTHH:MM:SS)."
+                "Invalid 'test_start_time' format. Provide a valid epoch time."
             )
```

Comment:
we should not expect the `test_start_time` field it should use the datetime.now rather.